server/app.ts

```diff
@@ -1,22 +1,34 @@
 import express, { Request, Response } from 'express';
+import morgan from 'morgan';
```

Comment:
Heroku won't be able to import morgan as a dev dependency, So try to import it conditionally